package.json

```diff
@@ -56,13 +56,6 @@
     "reselect": "^4.1.5",
     "url": "^0.11.0",
     "web-vitals": "^2.1.0",
-    "workbox-cacheable-response": "^6.5.4",
-    "workbox-core": "^6.5.4",
-    "workbox-expiration": "^6.5.4",
-    "workbox-precaching": "^6.5.4",
-    "workbox-routing": "^6.5.4",
-    "workbox-strategies": "^6.5.4",
-    "workbox-window": "^6.5.4",
```

Comment:
why you removed these?

Reply:
What is this for? Actually, Tapas asked me if these were added by me, which were not. So I thought I'll remove them. Were these used in your project?